dark/forms/tournament/add.py

```diff
@@ -13,3 +14,8 @@ class Meta:
         }
 
     number_of_teams = IntegerField()
+
+    def clean(self):
+        if self.cleaned_data.get('start_date') > self.cleaned_data.get('end_date'):
+            self.add_error('start_date', 'Start date must be before end date')
+            raise ValidationError('Start date must be before end date')
```

Comment:
Tą linijke kodu raczej trzebausunąć, bo `self.add_error('start_date', 'Start date must be before end date') wypisuje błąd konkretnego field'u, a `raise ValidationError('Start date must be before end date')` wypisuje błąd pod całym formularzem. Chociaż i widzimy ze strony frontu błąd tylko raz (bo nie wyświetlamy błędów drugiego rodzaju), ale on jest wysyłany dwa razy i jak w przyszłości będziemy chcieli wyświetlać dodatkowo drugi rodzaj błędów, to może to spowodować zamieszanie